.chglog/config.yml

```diff
@@ -0,0 +1,27 @@
+style: github
+template: CHANGELOG.tpl.md
+info:
+  title: CHANGELOG
+  repository_url: https://github.com/rancher-sandbox/cOS-toolkit
+options:
+  commits:
+    # filters:
+    #   Type:
+    #     - feat
+    #     - fix
+    #     - perf
+    #     - refactor
+  commit_groups:
+     title_maps:
+       feat: Features
+       fix: Bug Fixes
+       perf: Performance Improvements
+       refactor: Code Refactoring
+       ci: Continous Integration
```

Comment:
or maybe we should just use `group: commit message` instead from now on :)

Reply:
Those are not really used at the moment, until we switch to a specific commit style. I've left those in the config, so in case we choose a style, is just a matter of changing the template